release-team/role-handbooks/docs/README.md

```diff
@@ -58,6 +70,26 @@ General time requirements for leads and shadows are:
 - Between 1 and 2 hours a week to attend the majority of Release Team (weekly) and Burndown meetings (daily during Code Freeze), subject to time zone appropriateness
 - Up to 1 hour weekly to attend [SIG Docs meetings](https://github.com/kubernetes/community/tree/master/sig-docs#meetings) for status reports
 
+#### Onboarding Session (week 1) ~1 hour
+
+In the first week of the release cycle, the Docs Lead will organize an onboarding session with the shadows to go over general responsibilities and expectations.
```

Comment:
Should we include going over krel as part of the onboarding session? 

Also @satyampsoni was creating a assignment sheet useful or is this something we should drop? https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-notes#week-0